src/elements/common/messages.js

```diff
@@ -417,6 +417,11 @@ const messages = defineMessages({
         description: 'Title for the preview activity feed.',
         defaultMessage: 'Activity',
     },
+    sidebarDocgenTitle: {
+        id: 'be.sidebarDocgenTitle',
+        description: 'Title for the DocGen sidebar tab.',
+        defaultMessage: 'Doc Gen Tags',
```

Comment:
interesting, is this copy in the design? would normal users know these abbreviations?

Reply:
yes this is the name of the product